lib/qbuilder.js

```diff
@@ -0,0 +1,763 @@
+const Assert = require('assert')
```

Comment:
is this used only by postgres-store?

Reply:
This flavor of the `QBuilder` is only used by the `postgres-store`. The `mysql-store` uses a different flavor of it. For now, we can think of this module as a set of utilities that are private to the store plugin.